src/coredata.cs

```diff
@@ -1818,8 +2027,24 @@ partial interface NSPersistentStoreCoordinator
 		[NoWatch][NoTV]
 		[Since (7,0), Mavericks]
 		[Field ("NSPersistentStoreUbiquitousContainerIdentifierKey")]
+		[Obsolete ("Use UbiquitousContainerIdentifierKey instead.")]
 		NSString eUbiquitousContainerIdentifierKey { get; }
 
+		[NoWatch][NoTV]
+		[Since (7,0), Mavericks]
+		[Field ("NSPersistentStoreUbiquitousContainerIdentifierKey")]
+		NSString UbiquitousContainerIdentifierKey { get; }
```

Comment:
Do you want me to add an intro test that makes sure that the Fields are correctly expelled? Or to add it for this one in particular?